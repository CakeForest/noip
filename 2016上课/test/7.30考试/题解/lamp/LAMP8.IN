400
0 200 200 10000
1 7 0 1069
2 8 0 3341
3 9 0 875
4 1 0 9000
5 6 0 8753
6 2 0 849
7 3 0 5259
8 0 0 7747
9 6 0 5621
10 5 0 4380
11 9 0 6444
12 6 0 3363
13 5 0 488
14 4 0 1653
15 5 0 4315
16 0 0 6698
17 3 0 3142
18 1 0 8627
19 9 0 2167
20 9 0 2835
21 6 0 533
22 4 0 1515
23 2 0 5230
24 2 0 5167
25 4 0 1644
26 7 0 2589
27 0 0 7366
28 8 0 7561
29 3 0 6665
30 2 0 5594
31 4 0 8078
32 7 0 9177
33 1 0 282
34 5 0 9082
35 9 0 743
36 6 0 706
37 4 0 8453
38 1 0 1693
39 2 0 289
40 1 0 9311
41 1 0 6203
42 1 0 996
43 4 0 4110
44 6 0 5776
45 7 0 5123
46 3 0 9859
47 2 0 3190
48 2 0 6501
49 8 0 301
50 8 0 3894
51 5 0 7695
52 8 0 7153
53 0 0 8454
54 9 0 9967
55 3 0 3992
56 4 0 9359
57 1 0 8228
58 1 0 4596
59 2 0 353
60 8 0 20
61 4 0 3415
62 8 0 6489
63 3 0 4201
64 0 0 4827
65 1 0 481
66 3 0 5252
67 8 0 8915
68 4 0 1025
69 1 0 7462
70 4 0 6111
71 8 0 770
72 0 0 782
73 1 0 9723
74 0 0 1994
75 6 0 2708
76 3 0 3965
77 6 0 4091
78 0 0 5018
79 6 0 422
80 7 0 1182
81 1 0 4146
82 1 0 9404
83 4 0 3312
84 2 0 3265
85 1 0 5900
86 2 0 9865
87 7 0 3597
88 0 0 8900
89 5 0 9117
90 5 0 9850
91 4 0 3988
92 8 0 5345
93 2 0 485
94 7 0 3107
95 9 0 1246
96 5 0 2725
97 4 0 2832
98 8 0 1868
99 2 0 1881
100 7 0 676
101 3 0 4754
102 5 0 2086
103 0 0 4266
104 5 0 2389
105 1 0 7842
106 8 0 1152
107 5 0 5228
108 9 0 7481
109 5 0 3527
110 0 0 5071
111 8 0 9103
112 0 0 9570
113 1 0 9832
114 9 0 4637
115 2 0 921
116 2 0 1671
117 4 0 1045
118 7 0 3811
119 7 0 5293
120 9 0 5685
121 1 0 8120
122 6 0 288
123 7 0 4412
124 1 0 7339
125 3 0 946
126 1 0 8136
127 8 0 8065
128 5 0 5656
129 5 0 2585
130 9 0 397
131 0 0 5713
132 9 0 1593
133 1 0 1394
134 3 0 815
135 4 0 252
136 8 0 3472
137 9 0 7952
138 5 0 2499
139 4 0 3585
140 4 0 6657
141 9 0 9218
142 6 0 3580
143 1 0 6114
144 9 0 9303
145 8 0 6322
146 6 0 3870
147 9 0 4095
148 5 0 8804
149 0 0 3980
150 8 0 9336
151 5 0 8882
152 3 0 3069
153 5 0 5247
154 9 0 2316
155 4 0 4697
156 1 0 6005
157 9 0 6081
158 0 0 7849
159 4 0 6637
160 2 0 3517
161 3 0 3035
162 6 0 8764
163 3 0 8932
164 0 0 3533
165 8 0 6534
166 1 0 5048
167 6 0 7624
168 2 0 2851
169 2 0 5828
170 5 0 5162
171 6 0 7158
172 6 0 3406
173 5 0 8224
174 8 0 5324
175 2 0 7910
176 9 0 8277
177 9 0 1505
178 6 0 9792
179 6 0 3925
180 1 0 548
181 2 0 8421
182 7 0 4069
183 6 0 5185
184 2 0 162
185 8 0 1051
186 7 0 5208
187 9 0 8122
188 6 0 9651
189 7 0 3967
190 0 0 3075
191 5 0 2746
192 6 0 5744
193 1 0 5602
194 9 0 1319
195 4 0 6638
196 9 0 5214
197 6 0 2864
198 6 0 7947
199 3 0 1475
200 7 0 2489
201 4 0 1865
202 4 0 8358
203 5 0 4656
204 3 0 3917
205 1 0 6753
206 8 0 6226
207 4 0 7309
208 9 0 7354
209 3 0 5668
210 3 0 1941
211 2 0 1393
212 5 0 1800
213 8 0 327
214 7 0 1839
215 1 0 3447
216 4 0 2282
217 7 0 384
218 5 0 1823
219 3 0 5950
220 3 0 6337
221 5 0 6999
222 7 0 5214
223 8 0 6559
224 5 0 3078
225 8 0 8449
226 4 0 249
227 0 0 4047
228 9 0 3126
229 3 0 5991
230 9 0 3667
231 4 0 6474
232 3 0 3769
233 7 0 6081
234 5 0 4537
235 5 0 4086
236 2 0 7039
237 1 0 5453
238 0 0 778
239 8 0 1812
240 1 0 594
241 2 0 2727
242 0 0 511
243 2 0 2023
244 1 0 8119
245 7 0 2386
246 1 0 1540
247 9 0 9483
248 9 0 623
249 7 0 2183
250 1 0 7551
251 9 0 7677
252 3 0 429
253 8 0 319
254 9 0 53
255 4 0 9355
256 6 0 1958
257 0 0 6195
258 9 0 6192
259 5 0 7018
260 4 0 6796
261 4 0 4437
262 4 0 7185
263 8 0 5184
264 6 0 2625
265 0 0 7031
266 9 0 7191
267 6 0 3824
268 5 0 8204
269 7 0 1197
270 2 0 5375
271 4 0 1040
272 5 0 2326
273 4 0 5637
274 5 0 4639
275 3 0 4813
276 1 0 9212
277 2 0 125
278 7 0 732
279 2 0 5119
280 2 0 4576
281 9 0 7032
282 8 0 7129
283 1 0 8347
284 4 0 6044
285 3 0 1649
286 5 0 1092
287 1 0 1749
288 8 0 4790
289 9 0 976
290 1 0 7058
291 7 0 680
292 5 0 1943
293 6 0 4184
294 4 0 5273
295 2 0 2648
296 7 0 2257
297 4 0 9444
298 1 0 7267
299 3 0 1722
300 1 0 1543
301 7 0 8397
302 2 0 5522
303 4 0 4786
304 9 0 2340
305 5 0 8647
306 0 0 9336
307 7 0 3879
308 0 0 6205
309 0 0 2
310 1 0 6382
311 1 0 7354
312 6 0 3502
313 1 0 3410
314 3 0 8407
315 1 0 5894
316 8 0 2455
317 2 0 6201
318 5 0 3460
319 8 0 6544
320 5 0 6515
321 1 0 7630
322 6 0 9987
323 1 0 2445
324 5 0 531
325 6 0 2799
326 8 0 9335
327 7 0 8433
328 0 0 1456
329 2 0 867
330 2 0 7603
331 2 0 6639
332 7 0 3980
333 7 0 1312
334 5 0 8508
335 9 0 4278
336 1 0 6815
337 2 0 7302
338 8 0 90
339 8 0 813
340 1 0 2454
341 1 0 9193
342 8 0 8824
343 1 0 1446
344 5 0 8712
345 9 0 3615
346 0 0 546
347 3 0 3272
348 6 0 1587
349 9 0 8068
350 0 0 8648
351 1 0 6399
352 8 0 4607
353 8 0 6052
354 4 0 4328
355 4 0 5611
356 4 0 6504
357 0 0 5474
358 8 0 8553
359 9 0 5477
360 9 0 54
361 9 0 3902
362 6 0 353
363 4 0 7385
364 4 0 3484
365 6 0 4535
366 1 0 7293
367 2 0 2563
368 1 0 3044
369 8 0 4679
370 2 0 2624
371 3 0 8756
372 6 0 160
373 2 0 1412
374 4 0 3559
375 3 0 4223
376 3 0 1494
377 3 0 8916
378 7 0 9721
379 5 0 5611
380 6 0 7955
381 5 0 4750
382 2 0 708
383 6 0 6204
384 1 0 4186
385 3 0 7822
386 2 0 4526
387 6 0 5340
388 5 0 293
389 5 0 6246
390 6 0 7988
391 5 0 4131
392 7 0 4011
393 4 0 9788
394 9 0 3371
395 4 0 3957
396 5 0 4026
397 5 0 4625
398 9 0 4182
399 5 0 3942
