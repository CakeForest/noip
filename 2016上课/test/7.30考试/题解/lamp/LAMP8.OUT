2 11069
1 2
